### PR TITLE
Fix a few wording issues

### PR DESCRIPTION
**Pull request by skade**
As the title says, just a few errors that occured to me.

Thanks for providing the plugin!

### CODE REVIEW

File: _docs/faq.markdown [hunk 1]
```diff
@@ -3,17 +3,17 @@ title: FAQ
...
+{% include h title="Where is the Rust SDK?" tag="sdk" %}
 
 We don't use IntelliJ IDEA's SDK concept to manage Rust versions because it is
-specific to IDEA and is not present, for example, in the CLion. Path to Cargo is
-configured per project in the `Settings > Languages & Frameworks > Rust`.
+specific to IDEA and is not present, for example, in CLion. The path to Cargo is
+configured per project through `Settings > Languages & Frameworks > Rust`.
 
-{% include h title="Why Make Project action does not work?" tag="make" %}
+{% include h title="Why doe the Make Project action not work?" tag="make" %}
```
Comment (mkaput) [inline _docs/faq.markdown]: `does`

File: _docs/faq.markdown [hunk 2]
```diff
@@ -3,17 +3,17 @@ title: FAQ
 order: 999999
 ---
 
-{% include h title="Where is Rust SDK?" tag="sdk" %}
+{% include h title="Where is the Rust SDK?" tag="sdk" %}
```
Comment (mkaput) [inline _docs/faq.markdown]: I am never sure whether to use or not to use articles in titles, but I think they are rather omitted. Also personally it feels better to me without this `the` 😄

File: _docs/faq.markdown (hunk 1 above)
Comment (skade) [inline _docs/faq.markdown]: Heh :D. I amended the patch.

File: _docs/faq.markdown (hunk 2 above)
Comment (skade) [inline _docs/faq.markdown]: As a fellow non-native speaker, I understand the feeling, but that's definitely not the right grammar :D.

File: _docs/faq.markdown [hunk 3]
```diff
@@ -3,17 +3,17 @@ title: FAQ
...
 
 We don't use IntelliJ IDEA's SDK concept to manage Rust versions because it is
-specific to IDEA and is not present, for example, in the CLion. Path to Cargo is
-configured per project in the `Settings > Languages & Frameworks > Rust`.
+specific to IDEA and is not present, for example, in CLion. The path to Cargo is
+configured per project through `Settings > Languages & Frameworks > Rust`.
 
-{% include h title="Why Make Project action does not work?" tag="make" %}
+{% include h title="Why does the Make Project action not work?" tag="make" %}
```
Comment (matklad) [inline _docs/faq.markdown]: BTW, this action just works in the latest release, and you can even configure `cargo check` for it :) !

**Comment by matklad** [comment]
Thanks a lot @skade !

**Pull request merged**